### ISSUE TITLE
Language of page (text) is not set programmatically - Accessibility

### ISSUE THREAD

**Issue by brichwin** [h5p/h5p-multi-choice#76]
The containing html element does not have a lang attribute with appropriate value to indicate the default language of the text in the H5P widget. 

Please see https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/ for an excellent explanation of the concern. 

The related WCAG 2.0/2.1 Success Criteria is SC 3.1.1 (Level A): https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html

I suggest using the selected translation language to set a lang attribute on the html element to the appropriate value. See https://www.w3.org/WAI/WCAG21/Techniques/html/H57 for more info.

This is especially important in the context of using H5P in higher education eTexts as many have large populations of international students.

Thanks!
